boards/tenstorrent/tt_blackhole/tt_blackhole_fixed_partitions.dtsi

```diff
@@ -14,101 +20,173 @@ partitions {
-		label = "ethsdreg";
-		reg = <0x46000 DT_SIZE_K(4)>;
-		binary-path = "$BLOBS_DIR/tt_blackhole_serdes_eth_fwreg.bin";
+	/* Remote chip firmware images */
+	ethfw: partition@db000 {
+		label = "ethfw";
+		reg = <0xdb000 DT_SIZE_K(64)>;
+		binary-path = "$BLOBS_DIR/tt_blackhole_erisc.bin";
+		read-only;
+	};
+
+	memfw: partition@eb000 {
+		label = "memfw";
+		reg = <0xeb000 DT_SIZE_K(64)>;
+		binary-path = "$BLOBS_DIR/tt_blackhole_gddr_init.bin";
 		read-only;
 	};
 
-	ethsdfw: partition@47000 {
+	ethsdfw: partition@fb000 {
 		label = "ethsdfw";
-		reg = <0x47000 DT_SIZE_K(20)>;
+		reg = <0xfb000 DT_SIZE_K(64)>;
 		binary-path = "$BLOBS_DIR/tt_blackhole_serdes_eth_fw.bin";
 		read-only;
 	};
 
-	bmfw: partition@4c000 {
-		label = "bmfw";
-		reg = <0x4c000 DT_SIZE_K(68)>;
-		binary-path = "$BUILD_DIR/dmc/zephyr/zephyr.signed.bin";
-		read-only;
+	/* Update firmware image for the DMC */
+	dmfw: partition@10b000 {
+		label = "dmfw";
+		reg = <0x10b000 DT_SIZE_K(452)>;
+		binary-path = "$BUILD_DIR/dmc/zephyr/zephyr.bin";
 	};
 
-	flshinfo: partition@59000 {
-		label = "flshinfo";
-		reg = <0x59000 DT_SIZE_K(4)>;
-		binary-path =
-			"$BUILD_DIR/smc/generated_board_cfg/$BOARD_REV/flash_info.bin";
-		read-only;
+	/*
+	 * Image trailer for the DMC firmware update. Indicates to MCUBoot that
+	 * the image is valid and should be booted in test mode.
+	 */
+	dmfw_trailer: partition@17b000 {
+		label = "dmfw_trailer";
+		reg = <0x17b000 DT_SIZE_K(4)>;
+		binary-path = "$BLOBS_DIR/mcuboot_magic_test.bin";
 	};
 
-	boardcfg: partition@5a000 {
-		label = "boardcfg";
-		reg = <0x5a000 DT_SIZE_K(4)>;
-		binary-path = "$BUILD_DIR/smc/generated_board_cfg/$BOARD_REV/read_only.bin";
-		read-only;
+	/*
+	 * This partition is used for the primary firmware image.
+	 * it is loaded by MCUBoot. It is tagged as
+	 * mainimg in tt-boot-fs, but isn't used by the bootrom.
+	 */
+	slot0_partition: partition@17c000 {
+		label = "mainimg";
+		reg = <0x17c000 DT_SIZE_K(512)>;
+		binary-path = "$BUILD_DIR/smc/zephyr/zephyr.signed.bin";
 	};
 
 	/*
-	 * FIXME: The address of this node triggers a warning that we should try to fix:
-	 * Warning (unique_unit_address_if_enabled):
-	 *  /soc/spi@80070000/eeprom@0/partitions/partition@10000000
-	 * duplicate unit-address (also used in node
-	 *   /soc/spi@80070000/eeprom@0/partitions/failover_partition@10000000)
+	 * This partition defines a binary blob that indicates the
+	 * main image should be booted in test mode by mcuboot.
 	 */
-	failover: failover_partition@10000000 {
-		label = "failover";
-		reg = <0x10000000 DT_SIZE_K(76)>;
-		binary-path = "$BUILD_DIR/recovery/zephyr/zephyr.bin";
-		read-only;
+	main_test: partition@1fb000{
+		label = "maintail";
+		reg = <0x1fb000 DT_SIZE_K(4)>;
+		binary-path = "$BLOBS_DIR/mcuboot_magic_test.bin";
+	};
+
+	/* Leave some space for future fixed partitions, if needed. */
+
+	/* Free space for storage */
+	storage: partition@300000 {
+		label = "storage";
+		reg = <0x300000 DT_SIZE_M(61)>;
```

Comment:
```shell
echo "$((80 + 128 + 512 + 4 + 128 + 4 + 4 + 4 + 4 + 4 + 4 + 4 + 64 + 64 + 64 + 452 + 4 + 512 + 4))"
2044
```

64*1024 - 2044 = 63492, and 63492 / 1024 is 62, So I guess, technically, the storage partition could be 62 MiB.